nrf_security/include/mbedcrypto_glue/mbedtls/platform_alt.h

```diff
@@ -1,53 +1,101 @@
+*                                                                                     *
+* 3) Redistribution and use is permitted solely for the purpose of                    *
+*    developing or executing applications that are targeted for use                   *
+*    on an ARM-based product.                                                         *
+*                                                                                     *
+* DISCLAIMER. THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND                  *
+* CONTRIBUTORS "AS IS." ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT             *
+* NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY, NON-INFRINGEMENT,        *
+* AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE          *
+* COPYRIGHT HOLDERS OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,   *
+* SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED            *
+* TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR              *
+* PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF              *
+* LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING                *
+* NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS                  *
+* SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.                        *
+**************************************************************************************/
 
-/**@file
- * @defgroup mbedcrypto_glue mbed TLS glue layer
- * @{
- * @brief The mbedcrypto glue is a crypto toolbox glue layer coupling
- *        software and hardware-accelerated cryptography. The glue layer is implemented 
- *        using the mbed TLS ALT layer (also known as mbed TLS backend) and 
- *        is usable through regluar mbed TLS front ends.
- *
- * @note    If the functionality does not benefit from hardware-accelerated implementation
- *          or if it is not supported, the default behavior is to use the standard mbed TLS.
- * @}
- * @defgroup mbedcrypto_glue_platform_alt mbedcrypto platform glue
- * @ingroup mbedcrypto_glue
- * @{
- * @brief This is a glue layer for mbedcrypto platform abstraction.
- */
-#ifndef PLATFORM_ALT_H
-#define PLATFORM_ALT_H
+#ifndef MBEDTLS_PLATFORM_ALT_H
+#define MBEDTLS_PLATFORM_ALT_H
 
+#include <stddef.h>
 #include <stdint.h>
 
-#if !defined(MBEDTLS_CONFIG_FILE)
-#include "config.h"
-#else
+#if defined(MBEDTLS_CONFIG_FILE)
 #include MBEDTLS_CONFIG_FILE
 #endif
 
-#define CC310_MBEDTLS_PLATFORM_CONTEXT_WORDS    (6)//!<  Context size of of type used to initialize mbed TLS in words in the mbed_cc310_mbedcrypto library.
+#if defined(MBEDTLS_PLATFORM_SETUP_TEARDOWN_ALT)
+
+#define MBEDTLS_RND_WORKBUFFER_INTERNAL_SIZE    (1528)
+
+// These error codes are subject to change
+#define MBEDTLS_ERR_PLATFORM_PARAM_NULL	        -0x7001
+#define MBEDTLS_ERR_PLATFORM_INTERNAL	        -0x7002
```

Comment:
nit: indentation(but maybe GitHub UI which displays it weird.)

Reply:
fixed